core/platform/email/gae_email_services_test.py

```diff
@@ -38,8 +44,166 @@ def test_sending_email(self):
 
         with self.swap(feconf, 'CAN_SEND_EMAILS', True):
             gae_email_services.send_mail(
-                feconf.SYSTEM_EMAIL_ADDRESS, feconf.ADMIN_EMAIL_ADDRESS,
+                feconf.SYSTEM_EMAIL_ADDRESS,
+                feconf.ADMIN_EMAIL_ADDRESS,
                 'subject', 'body', 'html', bcc_admin=False)
         messages = self.mail_stub.get_sent_messages(
             to=feconf.ADMIN_EMAIL_ADDRESS)
         self.assertEqual(1, len(messages))
+
+    def test_that_email_bcc_is_sent_if_bcc_admin_is_true(self):
+        with self.swap(feconf, 'CAN_SEND_EMAILS', True):
```

Comment:
Before the action you should check that get_sent_messages() returns 0.

Ditto below. See the [guidelines for writing backend tests](https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests).

Reply:
Done!

Reply:
@siddhantkhandelwal, is this resolved?

Reply:
Yes, I have added the necessary Baseline verifications!